packages/addons/service/snapclient/source/bin/snapclient.start

```diff
@@ -26,12 +26,14 @@ case "$LIBREELEC_ARCH" in
     ;;
 esac
 
+[ -n "$sc_ss" ] && sc_SS="--h $sc_ss"
```

Comment:
This is wrong - the option is either `-h` or `--host`, but not `--h`.

https://github.com/badaix/snapcast/blob/v0.15.0/client/snapclient.1#L34
https://manpages.ubuntu.com/manpages/disco/man1/snapclient.1.html